src/fragments/lib/storage/native_common/getting-started/common.mdx

```diff
@@ -126,5 +126,4 @@ Congratulations! You've uploaded a file to an s3 bucket. Check out the following
 - [List Files](/gen1/[platform]/build-a-backend/storage/list/)
 - [Remove files](/gen1/[platform]/build-a-backend/storage/remove/)
 - [File access levels](/gen1/[platform]/build-a-backend/storage/configure-access/)
-- [Using Lambda Triggers](/gen1/[platform]/build-a-backend/storage/lambda-triggers/)
-{/* - [Escape Hatch](/gen1/[platform]/build-a-backend/storage/sdk/) */}
+- [Using Lambda Triggers](/gen1/[platform]/build-a-backend/storage/lambda-triggers/) {/* - [Escape Hatch](/gen1/[platform]/build-a-backend/storage/sdk/) */}
```

Comment:
From the PR, I think this was merged incorrectly. It looks like we need to remove more of the content on this page: https://github.com/aws-amplify/docs/pull/7279/files#diff-cc020e13c693866ba1fd716da242bbc70cd43d93e897d1dc22bb5226bbf82d70

Reply:
remaining lines have been removed